pkgs/development/python-modules/requests-aws4auth/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, requests }:
+buildPythonPackage rec {
+  pname = "requests-aws4auth";
+  version = "0.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0g52a1pm53aqkc9qb5q1m918c1qy6q47c1qz63p5ilynfbs3m5y9";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Please include a comment.

Reply:
The test hang. I'm not sure why. Good point about adding a comment. Will do.